src/keywords.cpp

```diff
@@ -83,6 +83,7 @@ static const chunk_tag_t keywords[] =
    { "and_eq",           CT_SASSIGN,       LANG_C | LANG_CPP                                                           },
    { "as",               CT_AS,            LANG_CS | LANG_VALA                                                         },
    { "asm",              CT_ASM,           LANG_C | LANG_CPP | LANG_D                                                  },
+   { "asm",              CT_PP_ASM,        LANG_ALL | FLAG_PP                                                          },
```

Comment:
I don't think this is available in all languages. Can we just start with C/C++ ?